db/seeds.rb

```diff
@@ -1,7 +1,19 @@
 # This file should contain all the record creation needed to seed the database with its default values.
 # The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
 #
-# Examples:
-#
-#   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
-#   Character.create(name: 'Luke', movie: movies.first)
+require 'faker'
+
+user = CreateAdminService.new.call
```

Comment:
check comment about this service https://github.com/tux255/sports_hub_portal/pull/2/files#r641578308

Reply:
checked and solved
removed secrets.yml and service completely.

Reply:
solved. Removed service.